src/Backend/Entities/Employee.java

```diff
@@ -1,7 +1,95 @@
 package Backend.Entities;
 
+import Backend.AttendanceManagment.AttendanceCommands.IAttendanceCommand;
+import Backend.AttendanceManagment.AttendanceCommands.MarkAttendance;
+import Backend.AttendanceManagment.AttendanceCommands.ViewAttendanceHistory;
+import Backend.AttendanceManagment.AttendanceManager;
+import Utilities.DataHandling.AttendanceFileHandler;
+import Utilities.DataHandling.IAttendanceDataHandler;
+
+/**
+     * This class represents an Employee in the company.
+     */
 public class Employee {
-    public static void main(String[] args) {
+    // Employee attributes
+    private final String name;
+    private final String position;
+    private final int id;
+    private final Department department;
+    private final AttendanceManager attendanceManager;
+
+    public Employee(int id) {
+        this.id = id;
+        attendanceManager = new AttendanceManager();
+        isSuperVisor();
+        IAttendanceDataHandler iAttendanceDataHandler = new AttendanceFileHandler();
```

Comment:
This constractor should take the ID. needs to be edited after the attendance manager edit.